docs/source/man.rst

```diff
@@ -690,6 +728,8 @@ Creating a new network type credential with a keyfile
   ``qpc cred add --name=new_creds --type=network --username=qpc_user --sshkeyfile=/etc/ssh/ssh_host_rsa_key``
 Creating a new network type credential with a password
   ``qpc cred add --name=other_creds --type=network --username=qpc_user_pass --password``
+Creating a new openshift type credential with a token
+  ``qpc cred add --name=other_creds2 --type=openshift --token``
```

Comment:
Shouldn't we also have an example showing adding type=ansible here too ?

Reply:
we dont have examples showing vcenter and satellite, I felt the need to add one for openshift because it doesnt follow the same pattern of the others. Do you think I should add one for each @abellotti ?

Reply:
It would not hurt to have examples for the others too. Thank you.

Reply:
agreed